tajo-cli/src/main/java/org/apache/tajo/cli/tsql/DefaultTajoCliOutputFormatter.java

```diff
@@ -206,4 +278,30 @@ public static String parseErrorMessage(String message) {
 
     return message;
   }
+
+  private static boolean detectRealTerminal()
```

Comment:
Yes. You're right. I've not known the rule about code redistribution. I'll add the comment explicitly.